src/configs/base.ts

```diff
@@ -269,6 +269,23 @@ export const config: Config = {
         figmaName: 'Border/Subdued',
       },
     },
+    {
+      name: 'borderDepressed',
```

Comment:
I think this name makes sense but open to something else. I also considered `borderStrong` but that didn't seem to fit with our other naming conventions

Reply:
`borderInsetShadow`? I am fine with the current naming for now.